mon-pix/app/styles/pages/_profilv2.scss

```diff
@@ -10,4 +10,9 @@
     padding-left: 30px;
     padding-right: 30px;
   }
+
+  &__link-to-compte {
+    display: flex;
+    justify-content: flex-end;
+  }
```

Comment:
J'ai remarqué que la classe `profilv2__link-to-compte` était imbriquée dans la classe `profilv2__panel`. Du coup, je pense qu'il faut plutôt nommer en `profilv2-panel__link-to-compte`